File: program/find-volume-of-a-cuboid/findVolumeOfACuboid.scala
object Main extends App {
  def volumeOfACuboid[T](wid: T, len: T, hei: T)(implicit ev: Numeric[T]) =
    ev.times(wid, ev.times(len, hei))

  val length = 2
  val width = 3
  val height = 4

  println("Length : " + length)
  println("Width  : " + width)
  println("Height : " + height)

  println("\nVolume : " + volumeOfACuboid(length, width, height))
}

File: program/program/find-volume-of-a-cuboid/find-volume-of-a-cuboid.py
def find_volume_of_a_cuboid(length, width, height):
    return length * width * height


if __name__ == "__main__":
    length = 2
    width = 3
    height = 4
    print("Length : ", length)
    print("Width  : ", width)
    print("Height : ", height)
    print("\nVolume : ", find_volume_of_a_cuboid(length, width, height))
